e2e-tests/ukef/cypress/support/portal/gefDealCancellationFlow.js

```diff
@@ -0,0 +1,42 @@
+const relative = require('../../e2e/relativeURL');
+const gefPages = require('../../../../gef/cypress/e2e/pages');
+const { PIM_USER_1, TFM_URL } = require('../../../../e2e-fixtures');
+const MOCK_USERS = require('../../../../e2e-fixtures/portal-users.fixture');
+
+const { BANK1_MAKER1, BANK1_CHECKER1 } = MOCK_USERS;
+
+module.exports = (dealId, effectiveDate) => {
```

Comment:
sorry to be a bit pedantic but a couple of things:
- I know lots of places use module.exports like this but I'd much prefer a named export, i.e.
```suggestion
const exportedThing = (dealId, effectiveDate) => {
  // ...

module.exports = {
  exportedThing
}
```